### PR TITLE
Add CRANE_PREFIX environment variable support.

### PR DESCRIPTION
**Pull request by tmc**

### CODE REVIEW

File: crane/cmd.go [hunk 1]
```diff
@@ -2,9 +2,10 @@ package crane
 
 import (
 	"fmt"
-	"github.com/alecthomas/kingpin"
 	"os"
 	"strings"
+
+	"github.com/alecthomas/kingpin"
```
Comment (michaelsauter) [inline crane/cmd.go]: Did you run that through gofmt? I'd expect no blank line here ...

**Comment by michaelsauter** [comment]
I like it! Good idea. Will merge once the formatting issue is addressed.

**Comment by bjaglin** [comment]
:+1: should we generalize that to other CLI flags for consistency? `CRANE_VERBOSE`, `CRANE_CONFIG`

File: crane/cmd.go (hunk 1 above)
Comment (tmc) [inline crane/cmd.go]: that's the output from goimports - https://godoc.org/golang.org/x/tools/cmd/goimports

Comment (michaelsauter) [inline crane/cmd.go]: Ah okay - then that's all good. I wanted to get that running a while back but for unclear reasons it did not work ... have to try that again, looks simple to setup.

**Comment by michaelsauter** [comment]
@bjaglin Sounds good to me. @tmc What do you think? Want to add them as well?

**Comment by tmc** [comment]
@bjaglin @michaelsauter yeah I think for consistency they should also be added.

**Pull request merged**